src/pretix/eventyay_common/navigation.py

```diff
@@ -105,6 +105,23 @@ def get_event_navigation(request: HttpRequest, event: Event) -> List[MenuItem]:
     # Merge plugin items into default navigation
```

Comment:
I think we do not want any plugin to show up here. Right now we have the plugins stated in the issue. Other plugins that are currently not installed are not mentioned in the issue, but we dont want them to show up in future neither.